<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadWindow</class>
 <widget class="QWidget" name="DownloadWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>116</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video Download</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblStatus">
     <property name="text">
      <string notr="true">[status]</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="prgFile">
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="format">
      <string>%v / %m bytes</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="prgTotal">
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="format">
      <string>%v / %m chunks</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="btnBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Abort|QDialogButtonBox::Cancel|QDialogButtonBox::Close|QDialogButtonBox::Open|QDialogButtonBox::Retry</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
